packages/components/src/lib/hooks/useCanvas.ts

```diff
@@ -30,20 +35,32 @@ export function useCanvas({
   const devicePixelRatio =
     typeof window === "undefined" ? 1 : window.devicePixelRatio;
 
+  const throttleTimeout = useRef<number | null>(null);
+
+  const handleResize = (entries: ResizeObserverEntry[]) => {
```

Comment:
This should be wrapped in `useCallback` in passed to observer's `useMemo` deps below, otherwise linter will (correctly) complain.

Reply:
Changed, let me know if that works.

Reply:
Yes. On the second thought, it might be easier to move `handleResize` inside `useMemo`, since no one else depends on it, but this approach is fine too.